include/z64pad.h

```diff
@@ -0,0 +1,14 @@
+#ifndef Z64PAD_H
+#define Z64PAD_H
```

Comment:
It would be convenient if this were named `padmgr.h` to mirror oot
(something something padmgr pr waiting room 👴)

Reply:
I had another look at this and putting `Input` in `padmgr.h` feels wrong because it should belong to padutils.
A future `padmgr.h` should have the `PadMgr` struct and functions and rely on whatever header `Input` is.
So I'll rename this header to `padutils.h` and let a future PR to cleanup PadMgr and its functions